src/Network/Network.Test/NrpTeamAlias.cs

```diff
@@ -49,5 +49,8 @@ class NrpTeamAlias
 
         //Azure NRP Firewall dev team
         public const string azurefirewall = "azurefirewall";
+
+        // Azure NRP bastion dev team
+        public const string bastion = "bastion";
```

Comment:
Looks like `bastion` team is reorged to NRP dev team. Should it remove from `Category`?

Reply:
I think we can remove all NRP aliases from `Category` since they are duplicates (if you're okay with that)

Reply:
Sure~I think you can remove them if there is not any reference any more.